src/main.tsx

```diff
@@ -1,39 +1,40 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
-import Layout from "./modules/Layout.tsx";
-import NotFound from "./pages/NotFound.tsx";
-import Homepage from "./pages/Homepage.tsx";
-import Login from "./pages/Login.tsx";
-import Canvas from "./pages/Canvas.tsx";
-import Signup from "./pages/Signup.tsx";
+import Layout from "./components/Layout/Layout.tsx";
```

Comment:
```suggestion
import Layout from "./components/Layout/Layout";
```
extension is not needed in import statement